master/buildbot/test/unit/test_changes_github.py

```diff
@@ -162,6 +239,11 @@ def test_SimplePR(self):
         self.assertEqual(change['repository'],
                          'https://github.com/defunkt/buildbot.git')
         self.assertEqual(change['files'], ['README.md'])
+        self.assertDictContainsSubset(_GH_PARSED_PROPS, change['properties'])
```

Comment:
this is deprecated in py3.2 (and we dont allow warnings)

https://docs.python.org/3/whatsnew/3.2.html?highlight=assertdictcontainssubset

you probably will need to replace that with a for k,v in _GH_PARSED_PROPS.items()